### PR TITLE
Add param for reporting limited namespace scoping feature

### PR DESCRIPTION
**Pull request by alicenoknow**
In order to have flexible control over namespaces access denial limit value, I've added it as a parameter to the CR. It will be passed as env var to license service.

Parent issue: https://github.ibm.com/IBMPrivateCloud/roadmap/issues/61869

### CODE REVIEW

File: api/v1alpha1/features.go [hunk 1]
```diff
@@ -50,6 +50,11 @@ type Features struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Custom namespaces Config Map",xDescriptors="urn:alm:descriptor:com.tectonic.ui:hidden"
 	// +optional
 	CustomNamespaceScopeConfigMap *string `json:"nssConfigMap,omitempty"`
+
+	// Namespace denial limit for custom namespaces scoping
```
Comment (kflorianski-ibm) [inline api/v1alpha1/features.go:54]: Please add to this docstring that this is the number of retrials

**Review by kflorianski-ibm — CHANGES_REQUESTED** [review]

File: api/v1alpha1/features.go (hunk 1 above)
Comment (kflorianski-ibm) [reply api/v1alpha1/features.go:54]: Remember to re-generate bundle after this change

File: api/v1alpha1/features.go [hunk 2]
```diff
@@ -50,6 +50,11 @@ type Features struct {
...
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:hidden"
```
Comment (Jan-Olszowka) [inline api/v1alpha1/features.go:55]: Should this field's descriptor be hidden on OCP UI? `descriptor:com.tectonic.ui:hidden` There will be no way to modify it from UI when creating CR.

Comment (alicenoknow) [reply api/v1alpha1/features.go:55]: Yes, that's intended. This field should be used only for testing purposes.

**Review by Jan-Olszowka — APPROVED** [review]

**Review by kflorianski-ibm — APPROVED** [review]

**Pull request merged**